config/default/test/check-app-settings-is-committed

```diff
@@ -15,7 +15,7 @@ EOF
 trap explain_error ERR
 
 echo '[check-app-settings-js-is-committed] Generating app_settings.json...'
-./node_modules/.bin/medic-conf --no-check compile-app-settings
+./node_modules/.bin/medic-conf --no-check --local compile-app-settings
```

Comment:
I think medic-conf should work without a check. I'll open a separate issue for that. The `--no-check` flag I believe doesn't do anything anymore. It became  `--check` when we switched to a library to parse args.

Reply:
I thiiink this code has already changes and you've reviewed an old version.

Reply:
I even added a comment above removing `--no-check` :D

Reply:
yeah I see it now.  I believe I got there by clicking the link from my email. A bit odd.